apps/shooting-clubs/acceptance/features/authority-details.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const steps = require('../../');
+
+module.exports = {};
+
+Feature('Authority details step');
+
+Before((
+  I,
+  authorityDetailsPage
+) => {
+  I.visitPage(authorityDetailsPage, steps)
+});
+
+Scenario('the correct form elements are present', (
+  I,
+  authorityDetailsPage
+) => {
+  I.seeElement(authorityDetailsPage.field.referenceNumber);
+});
+
+Scenario('an error appears if I submit the form with an empty field', (
+  I,
+  authorityDetailsPage
+) => {
+  I.submitForm();
+  I.seeErrors(authorityDetailsPage.field.referenceNumberGroup);
+});
+
+Scenario('I can go to the next page if I correctly fill the form', (
+  I,
+  authorityDetailsPage,
+  clubNamePage
+) => {
+  authorityDetailsPage.fillFormAndSubmit();
```

Comment:
Abstracting form filling logic is useful in cases where the same steps are repeated multiple times, but in cases like this I tend to prefer explicitly calling the steps within the feature file, otherwise we are adding unnecessary complexity for no gain. Also by including all logic here there is only one place to see all steps in the test